### ISSUE TITLE
Tune hyperparameters

### ISSUE THREAD

**Issue by s-broda** [s-broda/nmt#7]
We need to find a good balance between training time and model size. With the current ones, an epoch takes 40minutes. With those from the transformer paper, it takes 125 minutes. They train for 12h on 8 GPUs, which we can't really do; we only get 1 GPU, and I think we're limited to 120h per job on the cluster. I believe that the dictionary size might also play a big role. Not sure what the impact of that on training time is.

Note that with the round trip loss, training time will likely quadruple (because we need to translate de-en, en-de, and the do backprop twice more for the extra loss term). I think what we may want to do is train en-de and de-en separately first (two jobs on the cluster), then load the models and fine-tune with our new loss term. the separate models will only need to be trained once, so we can just let them run for 120h during the xmas break.
